include/libwalrus.hpp

```diff
@@ -16,8 +16,8 @@
 #include <eigenvalue_hafnian.hpp>
 #include <recursive_hafnian.hpp>
 #include <repeated_hafnian.hpp>
-#include <hafnian_approx.hpp>
-#include <torontonian.hpp>
+//#include <hafnian_approx.hpp>
+//#include <torontonian.hpp>

```

Comment:
How come these are commented out?

If they are no longer needed, should we just delete them?

Reply:
I got lazy. Will remove them together with the C++ files.